s/api/types/auth.ts

```diff
@@ -1,33 +1,34 @@
-
-import * as dbmage from "dbmage"
-import {AppDatabase} from "./schema.js"
+import * as dbmage from "dbmage";
+import { AppDatabase } from "./schema.js";
 
 export interface User {
-	userId: string
-	profile: Profile
-	permissions: {
-		canPost: boolean
-		canBanUsers: boolean
-		canDeleteAnyComment: boolean
-	}
+  userId: string;
+  profile: Profile;
+  permissions: {
+    canPost?: boolean;
```

Comment:
remove optionality for these permissions -- please remove the question marks, because no permissions should be "missing" when it's specified in the integration